src/components/tooltip/WrapTooltip.tsx

```diff
@@ -0,0 +1,25 @@
+import i18n from '@dhis2/d2-i18n'
+import { Tooltip, TooltipProps } from '@dhis2/ui'
+import React from 'react'
+
+type WrapWithTooltipProps = {
+    condition: boolean
+    children: React.ReactNode
+} & TooltipProps
+
+const TOOLTIPS = {
+    noEditAccess: i18n.t('You do not have access to edit this item.'),
+}
+
+export const WrapWithTooltip = ({
```

Comment:
the name and design of this component feels a bit weird for me - more like `TooltipWrapper` (to follow the rest of your naming convention). But also the tooltip message can be passed by the consumer along the condition - I don't see why it needs to be aware of "noEditAccess" for example.

Reply:
Updated now.